src/inverseRobot.js

```diff
@@ -7,7 +7,23 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const values = Object.values(robot);
+  const uniqueValues = [];
+  const inversedRobot = {};
+
+  for (const value of values) {
+    if (uniqueValues.indexOf(value) === -1) {
+      uniqueValues.push(value);
+    } else {
+      return null;
+    }
+  }
+
+  for (const key in robot) {
+    inversedRobot[robot[key]] = key;
+  }
```

Comment:
I suggest you remove this

You can assign reverse values in your first loop - that is more readable and more effective